packages/gatsby/cache-dir/loader.js

```diff
@@ -34,16 +34,23 @@ const loadPageDataJson = loadObj => {
   const url = createPageDataUrl(pagePath)
   return doFetch(url).then(req => {
     const { status, responseText } = req
+    let isJson = false
```

Comment:
NIT: this is never set to anything else and is used later in condition in https://github.com/gatsbyjs/gatsby/blob/482e4829f670c6bb910e4610b4db4e23b14dde1d/packages/gatsby/cache-dir/loader.js#L57 (there are no other usage of `isJson`) - we could remove this completely, right?

Reply:
that's true! thanks!